src/code/z_vr_box.c

```diff
@@ -236,8 +236,8 @@ void func_80143324(PlayState* play, SkyboxContext* skyboxCtx, s16 skyboxId) {
     size_t size;
 
     switch (skyboxId) {
-        case 1:
-            osCreateMesgQueue(&skyboxCtx->loadQueue, &skyboxCtx->loadMsg, 1);
+        case SKYBOX_NORMAL_SKY:
+            osCreateMesgQueue(&skyboxCtx->loadQueue, skyboxCtx->loadMsg, ARRAY_COUNT(skyboxCtx->loadMsg));
```

Comment:
fwiw OoT using `&loadMsg, 1` is a deliberate style choice, I think the idea is it's typically for single-use queues

Reply:
I kinda prefer the consistency of using ARRAY_COUNT everywhere, but I'm not that versed on this low level stuff... So I guess I'll revert this